Snippets/MsmqTransport/MsmqTransport_2/DisableInstaller.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NServiceBus;
+
+class DisableInstaller
+{
+    DisableInstaller(EndpointConfiguration endpointConfiguration)
+    {
+        #region disable-installer
+
+        var transport = new MsmqTransport
+        {
+            CreateQueues = false
```

Comment:
More of a general question rather than a comment on the code. Should `CreateQueues` (or maybe something like `CreateInfrastructure)` be on the `TransportDefinition` base class? And then being honored by all transports?

Reply:
we already bass a createInfrastructure flag on the host settings to the transport. Unfortunately this flag is currently tied to the installers in core so this flag is only needed when you want to disable queue creation while installers are enabled. Most other transports don't really have such an option at the moment and I think its a bit confusing anyway, but I kept it for backwards compatibility.